cypress/cypress.config.ts

```diff
@@ -0,0 +1,17 @@
+import { defineConfig } from 'cypress';
+import CypressVisualPlugin from '@saucelabs/cypress-visual-plugin';
+
+export default defineConfig({
+  e2e: {
+    saucelabs: {
+      buildName: 'SauceDemo - Cypress',
+      namePrefix: 'visual: ',
```

Comment:
The `namePrefix` isn't explained and for the sake of simplicity I would leave it out in the example.

Reply:
Agreed ! I removed it !